Feature/3.0.x new

kahvedigital opened the pull request:
Opencart 3.0.x Yeni Sürüm

kahvedigital committed:
Opencart 3.0 New Version

Opencart 3.0 New Version

kahvedigital committed:
Opencart 3.0 New 

Opencart 3.0 New

kahvedigital committed:
Opencart.3.0 New

Opencart.3.0 New

kahvedigital committed:
Opencart 3.0 New

Opencart 3.0 New

kahvedigital committed:
Opencart.3.0 New

Opencart.3.0 New

kahvedigital committed:
Sandbox silindi

Sandbox silindi

kahvedigital committed:
Sandbox silindi

Sandbox silindi

heheok commented on admin/controller/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,623 @@
+<?php
+
+error_reporting(E_ALL);
```
bu error reporting'in kapalı olması gerekiyor. server ile ilgili hassas bilgileri ekrana basma riski var.

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+// Entry
+$_['entry_api_id_live'] = 'Live Api Id';
+$_['entry_secret_key_live'] = 'Live Secret Key';
+$_['entry_order_status'] = 'Satın sonrası al Durum';
```
Bu çeviriler yanlış.  Tekrar elden geçirilmesi lazım. "Satın sonrası al durum" ne demek?

Çeviriyi google ile yapıyorsanız da sonrasında cümle ne anlama gelmeli, okuyunca ne anlıyorum diye bakmak lazım. Hepsine tek tek yorum yapmayacağım, zaten dosyayı baştan aşağı tarayınca problemli çevirilerin hepsi belli.

heheok commented on catalog/controller/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,788 @@
+<?php
+error_reporting(1);
```
error reporting'i development time'da debugging hariç kullanMAMAK gerekiyor.

heheok commented on catalog/language/en-gb/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,22 @@
+$_['text_title'] = 'Credit Card';
+$_['text_credit_card'] = 'Credit Card Details';
+$_['text_start_date'] = '(if available)';
+$_['text_issue'] = '(for Maestro and Solo cards only)';
```
SOLO card nedir?

heheok commented on catalog/language/en-gb/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,22 @@
+$_['text_title_in_checkout'] = 'Credit Card';
+// Entry
+$_['orderId_text'] = 'orderID';
+$_['Error_message_curl'] = 'You have to enable PHP CURL Extention on this server';
```
Bu son kullanıcının göreceği ekranda gösterilecek ise server'la ilgili detay son kullanıcıyı ilgilendirmediğinden hata mesajını değiştirmek gerekiyor.

heheok commented on catalog/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,22 @@
+$_['text_title_in_checkout'] = 'Kredi Kartı';
+// Entry
+$_['orderId_text'] = 'Sipariş no';
+$_['Error_message_curl'] = 'Sunucunuzda PHP CURL eklentisini etkinleştirmeniz gerekir';
```
Bu son kullanıcının göreceği ekranda gösterilecek ise server'la ilgili detay son kullanıcıyı ilgilendirmediğinden hata mesajını değiştirmek gerekiyor.

kahvedigital committed:
Update iyzico_checkout_form.php

Hata göstermeler kaldırıldı.

kahvedigital committed:
Update iyzico_checkout_form.php

Düzenlemeler yapıldı fakat daha önce bu çeviriler bizim tarafımızdan yapılmadı açıkcası hiç bir modülde bu geri dönüş olmadığı için çevirelerede dikkat etmemiştim diğer güncellemelerde bu çevirileri tekrardan gözden geçiririz.

kahvedigital committed:
Update iyzico_checkout_form.php

Hatta gösterme kodları kaldırıldı

kahvedigital committed:
Update iyzico_checkout_form.php

Debit olarak düzenlendi daha önce belirttiğim gibi dil dosyalarında hiç düzenleme yapılmadı.

kahvedigital committed:
Update iyzico_checkout_form.php

Bağlantı hatası şeklinde düzenlendi

kahvedigital committed:
Update iyzico_checkout_form.php

Bağlantı hatası şeklinde güncellendi

kahvedigital commented:
Değişiklikler güncellendi

kahvedigital committed:
Update iyzico_checkout_form.php

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+<?php
+
+// Heading
+$_['heading_title'] = 'Iyzico Ödeme Formu';
```
I küçük, her zaman...

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['heading_title'] = 'Iyzico Ödeme Formu';
+
+// Text 
+$_['text_link'] = 'iyzico pazar yeri sistemine kayıt olmak için';
```
"İyzico Üye İşyeri olmak için kayıt olun."

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['text_iyzico_checkout_form'] = '<a href="http://www.kahvedigital.com/" target="_blank"><img src="view/image/payment/iyzico_checkout_form.png" alt="iyzico" title="iyzico"/></a>';
+$_['text_iyzico_checkout_form_info'] = "Apı Id ve Secret Key anahtarlarınızı <a href='https://merchant.iyzipay.com/settings' target='_blank'>https://merchant.iyzipay.com/settings</a> 'dan alabilirsiniz.";
+// Entry
+$_['entry_api_id_live'] = 'Live Api Id';
```
"Canlı API ID"

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['text_iyzico_checkout_form_info'] = "Apı Id ve Secret Key anahtarlarınızı <a href='https://merchant.iyzipay.com/settings' target='_blank'>https://merchant.iyzipay.com/settings</a> 'dan alabilirsiniz.";
+// Entry
+$_['entry_api_id_live'] = 'Live Api Id';
+$_['entry_secret_key_live'] = 'Live Secret Key';
```
Canlı Secret Key

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['entry_status'] = 'Durumu';
+$_['entry_sort_order'] = 'Sıralama Türü';
+$_['entry_error_version'] = 'Yeni bir versiyon mevcut güncellemek için tıklayınız.';
+$_['entry_error_version_updated'] = 'Güncelleme Hatası! güncellemek için iyzico entegrasyon sayfasına giderek indirebilirsiniz. <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">Entegrasyon sayfası</a> ';
```
"Güncelleme Hatası! Manuel olarak güncellemek için <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">tıklayın</a>."

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+// Error
+$_['error_permission'] = 'Bu değişikliği yapmak için yetkiniz yok!';
+$_['error_api_id_live'] = 'API ID gereklidir!';
+$_['error_secret_key_live'] = 'Gizli anahtarı gereklidir!';
```
"Secret Key Gereklidir"

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['error_secret_key_live'] = 'Gizli anahtarı gereklidir!';
+$_['error_invalid_order'] = 'Geçersiz Sipariş';
+
+$_['iyzico_single_keys_must_configured'] = "Bu eklenti üzerinde değişiklik yapabilmeniz için eklendinin kurulu olması gereklidir";
```
"eklendinin" değil, "eklentinin"

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['iyzico_single_keys_must_configured'] = "Bu eklenti üzerinde değişiklik yapabilmeniz için eklendinin kurulu olması gereklidir";
+$_['Token Expired.'] = 'İşlem süresi doldu.'; 
+$_['Token is expired. Please try again.'] = 'İşlem süresi doldu. Lütfen tekrar deneyiniz.'; 
+$_['Invalid call.'] = 'Geçersiz arayın.';
```
"Geçersiz İstek"

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['Invalid call.'] = 'Geçersiz arayın.';
+
+// Details
+$_['payment_transaction_type'] = 'Ödeme İşlemi';
```
"Ödeme işlem tipi"

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['text_item_name'] = 'Ürün Adı';
+$_['text_paid_price'] = 'Tahsil Edilen Tutar';
+$_['text_total_refunded_amount'] = 'İade Edilen Tutar.';
+$_['text_action'] = 'Aksiyon';
```
"İşlem"

heheok commented on admin/language/tr-tr/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['text_type'] = 'Tip';
+$_['text_status'] = 'Durum';
+$_['text_note'] = 'Not';
+$_['text_are_you_sure'] = 'Bu işlemi yapmak için devam etmek istiyormusunuz?';
```
"Bu işlemi gerçekleştirmek istediğinizden emin misiniz?"

heheok commented on admin/language/en-gb/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+<?php
+
+// Heading
+$_['heading_title'] = 'Iyzico Checkout Form';
```
"I" her zaman küçük

heheok commented on admin/language/en-gb/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['entry_status'] = 'Status';
+$_['entry_sort_order'] = 'Sort Order';
+$_['entry_error_version'] = 'There is a new version. Click here to update.';
+$_['entry_error_version_updated'] = 'Update Error! go to iyzico version page <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">Version page</a> ';
```
"Update Error! <a href="https://www.iyzico.com/entegrasyon/hazir-altyapi/opencart/" target="_blank">Click here </a> to manually update the plugin.

heheok commented on admin/language/en-gb/extension/payment/iyzico_checkout_form.php:
```diff
@@ -0,0 +1,69 @@
+$_['cancel_done_success'] = 'Your order has been canceled successfully.';
+
+$_['request_amount_not_greater_than'] = 'Requested amount should not be greater than %s.';
+$_['refund_done_success'] = 'Refund done successfully. Item Id: %s, Item Name: %s, Amount: %s';
```
"Payment successfully refunded. Item Id: %s, Item Name: %s, Amount: %s"

heheok reviewed (changes requested):
Çevirilere daha fazla dikkat etmek gerekiyor... Aynı zamanda conflict var, README.md dosyası conflict veriyor, onun da düzeltilmesi lazım.

kahvedigital committed:
Update iyzico_checkout_form.php

kahvedigital committed:
Update iyzico_checkout_form.php

kahvedigital commented:
Değişiklikler yapıldı

heheok committed:
Merge branch '3.0.x' into feature/3.0.x-new
